Travel Scheduler/View Controllers/HomeCollectionViewController.m

```diff
@@ -220,17 +214,23 @@ - (NSIndexPath *)tableView:(UITableView *)tableView willSelectRowAtIndexPath:(NS
     moreOptionViewController.places = [[NSMutableArray alloc]init];
     moreOptionViewController.hub = self.hub;
     if (cellNum == 0) {
-        moreOptionViewController.stringType = @"Attractions";
-        NSMutableSet *set = [NSMutableSet setWithArray:self.hub.dictionaryOfArrayOfPlaces[@"museum"]];
-        [set addObjectsFromArray:self.hub.dictionaryOfArrayOfPlaces[@"park"]];
-        moreOptionViewController.places = (NSMutableArray *)[set allObjects];
+        moreOptionViewController.stringType = @"Parks";
+        moreOptionViewController.correctType = @"park";
+        moreOptionViewController.places = self.hub.dictionaryOfArrayOfPlaces[@"park"];
     } else if (cellNum == 1) {
-        moreOptionViewController.stringType = @"Restaurants";
+        moreOptionViewController.stringType = @"Museums";
+        moreOptionViewController.correctType = @"museum";
         moreOptionViewController.places = self.hub.dictionaryOfArrayOfPlaces[@"restaurant"];
     } else if (cellNum == 2) {
+        moreOptionViewController.stringType = @"Restaurants";
+        moreOptionViewController.correctType = @"restaurant";
+        moreOptionViewController.places = self.hub.dictionaryOfArrayOfPlaces[@"lodging"];
+    } else if (cellNum == 3) {
         moreOptionViewController.stringType = @"Hotels";
+        moreOptionViewController.correctType = @"lodging";
         moreOptionViewController.places = self.hub.dictionaryOfArrayOfPlaces[@"lodging"];
     }
+    moreOptionViewController.places = self.hub.dictionaryOfArrayOfPlaces[moreOptionViewController.correctType];
```

Comment:
You seem to be setting moreOptionViewController.places individually within all the if statements (and again here). I think you can remove all the lines inside the if statements to be less repetitive.

Reply:
omg I was planning on erasing that and I forgot! Thanks for noticing that!